### ISSUE TITLE
attend command

### ISSUE THREAD

**Issue by Trav1sT** [Trav1sT/ped#5]
steps:
1. attend 1 d/01/01/2020
2.attend 1 d/01/01/2020 output : nameB already attended on 01/01/2020
3. unattend 1 d/01/01/2020 output: nameB already unattended on 01/01/2020
4. showatt1 d/01/01/2020 Output: nameB was present on Wed Jan 01 00:00:00 SGT 2020
Expected: nameB was not present on Wed Jan 01 00:00:00 SGT 2020.

It is kind of bizarre to have person 1 to attend twice a day. I understand that it could be he attended class a then class b. If so then there should be a specifier for which class he is attedning. because based on the output of step 3 i would think he is "unattended" meaning he is offically marked as absent. if Unattend command only deletes one of the "attends", which one does it delete?

TLDR: It would be great if there can be different tutorial he can attend. If there is only 1 tutorial then unattending should mark him as absent.
